### ISSUE TITLE
json: cannot unmarshal string into Go value of type map[string]interface {}

### ISSUE THREAD

**Issue by joshuaspence** [terraform-providers/terraform-provider-aws#1441]
I am consistently see a crash with version 0.1.4 of `terraform-provider-aws`. I don't want to post the entire crash log as I am not sure if it contains sensitive information... but I think this is the relevant snippet:

```
2017/08/17 15:23:53 [DEBUG] apply: aws_launch_configuration.launch_configuration: executing Apply
2017-08-17T15:23:53.500+1000 [DEBUG] plugin.terraform-provider-aws_v0.1.4_x4: log from plugin: entry="2017/08/17 15:23:53 [DEBUG] Describing AMI "ami-fcdfc79f" to get root block device name"
2017-08-17T15:23:53.538+1000 [DEBUG] plugin.terraform-provider-aws_v0.1.4_x4: log from plugin: entry="2017/08/17 15:23:53 [INFO] Validating account ID"
2017-08-17T15:23:53.544+1000 [DEBUG] plugin.terraform-provider-aws_v0.1.4_x4: log from plugin: entry="2017/08/17 15:23:53 [INFO] Validating account ID"
2017-08-17T15:23:53.548+1000 [DEBUG] plugin.terraform-provider-aws_v0.1.4_x4: log from plugin: entry="2017/08/17 15:23:53 [DEBUG] autoscaling create launch configuration: {"
2017-08-17T15:23:53.548+1000 [DEBUG] plugin.terraform-provider-aws_v0.1.4_x4: log from plugin: entry="  EbsOptimized: false,"
2017-08-17T15:23:53.548+1000 [DEBUG] plugin.terraform-provider-aws_v0.1.4_x4: log from plugin: entry="  IamInstanceProfile: "InstanceProfile_gaf_worker","
2017-08-17T15:23:53.548+1000 [DEBUG] plugin.terraform-provider-aws_v0.1.4_x4: log from plugin: entry="  ImageId: "ami-fcdfc79f","
2017-08-17T15:23:53.548+1000 [DEBUG] plugin.terraform-provider-aws_v0.1.4_x4: log from plugin: entry="  InstanceMonitoring: {"
2017-08-17T15:23:53.548+1000 [DEBUG] plugin.terraform-provider-aws_v0.1.4_x4: log from plugin: entry="    Enabled: true"
2017-08-17T15:23:53.548+1000 [DEBUG] plugin.terraform-provider-aws_v0.1.4_x4: log from plugin: entry="  },"
2017-08-17T15:23:53.548+1000 [DEBUG] plugin.terraform-provider-aws_v0.1.4_x4: log from plugin: entry="  InstanceType: "r3.large","
2017-08-17T15:23:53.548+1000 [DEBUG] plugin.terraform-provider-aws_v0.1.4_x4: log from plugin: entry="  LaunchConfigurationName: "gaf_worker-0038b580badb0c1fa445bac1ed","
2017-08-17T15:23:53.548+1000 [DEBUG] plugin.terraform-provider-aws_v0.1.4_x4: log from plugin: entry="  SecurityGroups: ["
2017-08-17T15:23:53.548+1000 [DEBUG] plugin.terraform-provider-aws_v0.1.4_x4: log from plugin: entry="    "gaf_common-003eb284a95e2a6ca50b58f2df","
2017-08-17T15:23:53.548+1000 [DEBUG] plugin.terraform-provider-aws_v0.1.4_x4: log from plugin: entry="    "syd-dev","
2017-08-17T15:23:53.548+1000 [DEBUG] plugin.terraform-provider-aws_v0.1.4_x4: log from plugin: entry="    "base","
2017-08-17T15:23:53.548+1000 [ERROR] plugin.terraform-provider-aws_v0.1.4_x4: error parsing json from plugin: error="json: cannot unmarshal string into Go value of type map[string]interface {}"
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x48 pc=0xb7752c]

goroutine 18824 [running]:
github.com/hashicorp/terraform/vendor/github.com/hashicorp/go-plugin.(*Client).logStderr(0xc420a2c150, 0x3567a80, 0xc423525b60)
	/opt/gopath/src/github.com/hashicorp/terraform/vendor/github.com/hashicorp/go-plugin/client.go:750 +0x39c
created by github.com/hashicorp/terraform/vendor/github.com/hashicorp/go-plugin.(*Client).Start
	/opt/gopath/src/github.com/hashicorp/terraform/vendor/github.com/hashicorp/go-plugin/client.go:544 +0xe2f
```

Let me know if you require any further details.

**Comment by llavaud** [comment]
seems related to the issue i create yesterday #1429

**Comment by apparentlymart** [comment]
Hi @llavaud! Sorry for this crash.

I believe this is related to a bug in Core version v0.10.1, which was fixed in the patch release v0.10.2 yesterday. Could you give this a try with v0.10.2 and see if that fixes the problem?

**Comment by radeksimko** [comment]
👋 
I'm going to close this issue as it has been stale for nearly 2 months and it's very likely the issue was fixed as mentioned by @apparentlymart 

Do let us know if that's not the case.

Thanks for reporting it.
